test-data/unit/fine-grained.test

```diff
@@ -1001,6 +1001,37 @@ class A:
 ==
 main:2: error: Unsupported left operand type for < ("B")
 
+[case testAttrsUpdateKwOnly]
+[file a.py]
+import attr
+@attr.s(kw_only=True)
+class A:
+    a = attr.ib(15)   # type: int
+
+[file b.py]
+from a import A
+import attr
+@attr.s(kw_only=True)
+class B(A):
+    b = attr.ib("16")   # type: str
+
+B(b="foo", a=7)
+
+[file b.py.2]
+from a import A
+import attr
+@attr.s()
+class B(A):
+    b = attr.ib("16")   # type: str
+
+B(b="foo", a=7)
+
+[builtins fixtures/attr.pyi]
+[out]
+==
+b.py:5: error: Non keyword-only attributes are not allowed after a keyword-only attribute.
```

Comment:
This test doesn't test much of incremental logic. I would also add a test where base class in `a.py` is updated, while subclass is unchanged.